src/services/importing/conflicting-plugins-service.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Yoast\WP\SEO\Services\Importing;
+
+use Yoast\WP\SEO\Config\Conflicting_Plugins;
+
+/**
+ * Detects plugin conflicts.
+ */
+class Conflicting_Plugins_Service {
+
+	/**
+	 * Detects the conflicting plugins.
+	 *
+	 * @return array A list of all active conflicting plugins.
+	 */
+	public function detect_conflicting_plugins() {
+		$all_active_plugins = $this->get_active_plugins();
+
+		// Search for active plugins.
+		return $this->get_active_conflicting_plugins( $all_active_plugins );
+	}
+
+	/**
+	 * Deactivates the specified plugin(s) if any, or the entire list of known conflicting plugins.
+	 *
+	 * @param string|array|false $plugins (optional) The plugin filename, or array of plugin filenames, to deactivate.
+	 */
+	public function deactivate_conflicting_plugins( $plugins = false ) {
+		// If no plugins are specified, deactivate any known conflicting plugins that are active.
+		if ( ! $plugins ) {
+			$plugins = $this->detect_conflicting_plugins();
+		}
+
+		// In case of a single plugin, wrap it in an array.
+		if ( \is_string( $plugins ) ) {
+			$plugins = [ $plugins ];
+		}
+
+		if ( ! is_array( $plugins ) ) {
+			return;
+		}
+
+		// Deactivate all specified plugins across the network, while retaining their deactivation hook.
+		\deactivate_plugins( $plugins, false, true );
```

Comment:
The last argument `true` is for multisites only. This causes the function to do nothing on single-site installations. Negating the argument makes the function call work.